src/applications/mhv/landing-page/components/HubLinks.jsx

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import recordEvent from '~/platform/monitoring/record-event';
+import { getDestinationDomain } from '../utilities';
 
 const HubSection = ({ title, links }) => {
   const listItems = links.map(({ href, text }, index) => (
     <li key={`${href}--${index}`}>
       <va-link
         class="mhv-c-link"
-        disable-analytics
+        disable-analytics={{
```

Comment:
The [Design System storybook](https://design.va.gov/storybook/?path=/docs/components-va-modal--default) seems to indicate we had this right before, so I'd guess you heard otherwise? Did we create a ticket for the design system team to correct the documentation or fix [the implementation](https://github.com/department-of-veterans-affairs/component-library/blob/ce5ba5edab549d639872544d732147776bfa71ce/packages/web-components/src/components/va-link/va-link.tsx#L38)? 

I would guess its find to work around the issue, though would be a little worried that this might break at some point...

Reply:
I looked in the codebase and I couldn't find any any use of `disable-analytics`, everything I saw had `disable-analytics="false"` so I figured it was safer to follow that pattern.

Reply:
Press the "Show code" button, and toggle the `disable-analytics` property on the following page for an example of using `disable-analytics` without a value: https://design.va.gov/storybook/?path=/docs/components-va-link--default

The alternate way of doing this would be `disable-analytics={true}` or `disable-analytics={false}`, I believe.

Reply:
Since I am haven't had the pleasure of writing web components or using stencils, I took a look at [the Stencil docs on Boolean Props](https://stenciljs.com/docs/properties#boolean-props). It looks like the plain `disable-analytics` wouldn't work, but `disable-analytics="false"` would. 

I'm not sure how passing an object via JSX, as written in the code here, gets interpreted as a truth/falsey value. I think it would be clearer/safer to use `disable-analytics="true"` if we want to disable the web component's default analytics click handler